src/content/blog/dev-portal.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: "The Astro Developer Portal"
+description: "Today, we’re launching the Astro Developer Portal. A platform, where theme authors and creators, can fully create, own and update their themes built for Astro."
+publishDate: "March 13, 2024"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/dev-portal/dev-portal.webp"
+socialImage: "/src/content/blog/_images/dev-portal/dev-portal.webp"
+lang: "en"
+---
+
+import BlogContentImage from "/src/components/BlogContentImage.astro"
+import BlogSummary from "/src/components/BlogSummary.astro"
+import dashboard from "/src/content/blog/_images/dev-portal/dashboard.webp"
+
+<BlogSummary>
+Today, we're launching [the Astro Developer Portal](https://portal.astro.build). A platform, where theme authors and creators, can fully create, own and update their themes built for Astro.
+</BlogSummary>
+
+At Astro, we're dedicated to crafting solutions that not only benefit our users but also address our internal challenges. One such solution is the new Developer Portal, the new and future home for all Astro developers. Our aim with the Developer Portal is to provide a centralised platform where theme creators, and eventually integration creators, can have full control over their creations. Another goal was to test drive Astro Studio, its capabilities, and identify bugs early.
+
+Today, we're taking the new Astro Developer Portal in use. A platform, where theme authors and creators, can fully create, own and update their themes built for Astro.
+
```

Comment:
This conveys the same information as the blog summary presented just above, so I feel this can be removed

Reply:
yeah! I 100% agree. I'm searching for a way to make the "official announcement" here and then linking to the actual platform.
Just linking the platform on the line below alone, feels a bit out of place